library/python/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+cmake_minimum_required(VERSION 3.18)
+project(pywavemap LANGUAGES CXX)
+
+# Warn if the user invokes CMake directly
+if (NOT SKBUILD AND NOT $ENV{CLION_IDE})
+  message(WARNING "\
+  This CMake file is meant to be executed using 'scikit-build-core'.
```

Comment:
Does the 'scikit-build-core' bit need to change?

Reply:
Calling `pip install` invokes `scikit-build-core` to build wavemap, so this is correct - unless I misunderstood your question.

Reply:
No that's ok then, I wasn't sure if this was some left-over from guiding information on how to get this all running.